sdk-setup/src/sdk-manage

```diff
@@ -1063,9 +1121,23 @@ manage_object_packages() {
     fi
 
     object_zypper "$type:$object" --no-refresh "$action" "${packages[@]}" || return
-    if [[ $INSIDE_VBOX && $type == target ]] && ! target_is_snapshot "$object"; then
-        synchronise_target "$object"
+    maybe_synchronize_object "$type:$object"
+}
+
+maybe_synchronize_object() {
```

Comment:
Since we are already using the British spelling in synchronise_target, could we use it also here (i.e. maybe_synchronise_object)?

Reply:
yup :)